src/login-page/LoginMain.jsx

```diff
@@ -0,0 +1,107 @@
+import axios from "axios";
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { useAuth } from "../context/auth-context";
+import "./LoginPage.css";
+
+export default function LoginMain() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const { dispatch } = useAuth();
+  const navigate = useNavigate();
+
+  async function submitHandler(e) {
+    e.preventDefault();
+    console.log("clik");
+    console.log(email, password);
+
+    try {
+      const response = await axios.post("api/auth/login", {
+        email: email,
+        password: password,
+      });
+
+      if (response.status === 200 || response.status === 201) {
+        localStorage.setItem("authToken", response.data.encodedToken);
+        navigate("/");
+        dispatch({ type: "SET_AUTH", payload: true });
+      }
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  async function loginAsGuest() {
+    try {
+      const response = await axios.post("/api/auth/login", {
+        email: "adarshbalika@gmail.com",
+        password: "adarshBalika123",
+      });
+      console.log(response);
```

Comment:
**suggestion**: delete console logs if not needed in future

Reply:
sure